### ISSUE TITLE
Missing lots of stuff from various rule34 sites

### ISSUE THREAD

**Issue by zearmhot** [Bionus/imgbrd-grabber#1563]
Grabber is not downloading everything from shimmie Rule34 sites.

On rule34hentai, I put in a tag that has 1300 files. Grabber says ~530, and then after grabbing there is only 140 in the folder. Almost a 1/10th of what I see on the site.

Using the latest nightly if it matters.

**Comment by zearmhot** [comment]
Actually upon closer inspection, Rule34Hentai login no longer seems to work with grabber. It says successful but I get 120 results instead of 500+ (same as if I log out on chrome).

**Comment by zearmhot** [comment]
The sites working now but I still see way less in grabber than I do on rule34hentai. Tags with ~2000 images show up with ~600 results in grabber.

**Comment by zearmhot** [comment]
I found a workaround. I've been trying to wrap my head around this, it seems the pages display correctly but the image count grabber puts it very wrong. For example, on this tag I have 9 pages (~567 images) and I see all that correctly in grabber, but when I select get all. It only has 180 images, and only grabs about that much.

So after grabbing it, I set the image count to "99999". Then it grabs everything. This seems like a relatively simple fix, and affects every rule34 site I tested (rule34.xxx, rule34.pheal, etc)
